packages/components/breadcrumbs/src/breadcrumbs.scss

```diff
@@ -29,10 +30,26 @@ a,
 
 a,
 ::slotted(a) {
+  color: var(--_link-color);
   text-decoration: var(--_link-decoration);
   text-underline-offset: var(--_link-underline-offset, auto);
 }
 
+a:visited,
+::slotted(a:visited) {
+  --_link-color: var(--sl-color-href-idle-foreground);
+}
+
+a:hover,
+::slotted(a:hover) {
+  --_link-color: var(--sl-color-href-hover-foreground);
+}
+
+a:active,
+::slotted(a:active) {
+  --_link-color: var(--sl-color-href-active-foreground);
+}
+
```

Comment:
It looks like the outline with tokens is missing in focus-visible?

<img width="429" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/367637e0-2110-4834-b89a-0ca497250771">